### PR TITLE
Quiz 2

### PR DESCRIPTION
**Pull request by Jsharp517**
Created Quiz 2

### CODE REVIEW

File: level2.html [hunk 1]
```diff
@@ -15,6 +15,10 @@ <h1>Level 2</h1>
     <a href="index.html">
         <button>Click me to go home</button>
     </a>
+    <a href="Quiz2.html">
+        <button>Quiz 2</button>
+    </a>
+
```
Comment (ashleyplusplus) [inline level2.html]:
```suggestion
```
nit

**Review by ashleyplusplus — COMMENTED** [review]
just a whitespace nit

**Review by BethJ-20 — APPROVED** [review]
Changes look great and relatively easy  to follow

**Review by y1c1 — APPROVED** [review]
Looks good!

**Comment by Jsharp517** [comment]
Looks Good!!

**Pull request merged**